### PR TITLE
add README for Shopify Slack integration

### PR DESCRIPTION
**Pull request by chameerar**

### CODE REVIEW

File: integrations/ballerina/shopify-slack-order-notfication-service/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+1. **Admin API access token from Shopify store app:** Create an app in your Shopify store with `read-order` permission and save the  **Admin API access token**.
+2. **API secret key from Shopify store app:** Obtain an **API secret key** from the same app.
+3. **Bot OAuth token form Slack app:** Create a Slack app via [Slack API](https://api.slack.com/) with `channels:write`, `channels:read`, `chat:write`, and `chat:write.public` scopes and obtained the **Bot OAuth token**.
+
+A slack channel named **orders** should be created.
+
+## Deploying in Choreo
+
+1. **Fork the Repository:** Fork the [choreo-examples repository](https://github.com/wso2/choreo-examples.git).
```
Comment (ruks) [inline integrations/ballerina/shopify-slack-order-notfication-service/README.md:22]: Fork keyword is redundant right?

File: integrations/ballerina/shopify-slack-order-notfication-service/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+2. **Create Event-triggered integration Component in Choreo:** 
+- Log into Choreo, navigate to **Create a New Component** page in your Choreo Project and select **Event-Triggered Integration**.
+- Fill out general details; name and description.
```
Comment (ruks) [inline integrations/ballerina/shopify-slack-order-notfication-service/README.md:26]: do we need `;`?

File: integrations/ballerina/shopify-slack-order-notfication-service/README.md (hunk 1 above)
Comment (chameerar) [reply integrations/ballerina/shopify-slack-order-notfication-service/README.md]: Fixed in - 3b91eb4dcf51bb35b601e40564c9a01c623b52c4

File: integrations/ballerina/shopify-slack-order-notfication-service/README.md (hunk 2 above)
Comment (chameerar) [reply integrations/ballerina/shopify-slack-order-notfication-service/README.md]: Fixed in - 3b91eb4dcf51bb35b601e40564c9a01c623b52c4

**Review by a5anka — APPROVED** [review]

**Pull request merged**